Sources/AWSSDKSwiftCore/HTTP/NIOTSHTTPClient.swift

```diff
@@ -9,6 +9,7 @@
 // and the swift-server's swift-nio-http-client
 // https://github.com/swift-server/swift-nio-http-client
 //
+#if canImport(Network)
 
```

Comment:
Sadly I can't comment on line 20. The `#if canImport(Network)` can be removed.

Reply:
Yep should be removed, thought I had cleaned up those imports already. Oh well will do it again

Reply:
sorted